src/main/scala/akka/stream/alpakka/xlsx/Row.scala

```diff
@@ -17,7 +16,7 @@ final class Row(val rowIndex: Int, columnIndexToCell: TreeMap[Int, Cell]) {
     *         correspond to cells of adjacent columns. If a cell is missing in the `columnIndexToCell` mapping, a `Blank`
     *         cell is created with reference generated by `CellReference#generateReference`.
     */
-  def cells: Seq[Cell] = (1 to columnIndexToCell.keys.max).iterator
+  def cells: Seq[Cell] = (1 to Try(columnIndexToCell.keys.max).getOrElse(0)).iterator
     .map(columnIndex => columnIndexToCell.getOrElse(columnIndex, Cell.Blank(CellReference.generateReference(columnIndex, rowIndex))))
     .toSeq
```

Comment:
You can skip `.iterator` on the `Range` and remove this `.toSeq`. You'll still avoid intermediate collections and you'll get an `IndexedSeq` instead of a `Stream` as a bonus, which should be better for accessing cells later on - they're usually accessed via index.